testinfra/test_ami.py

```diff
@@ -0,0 +1,307 @@
+import base64
+import boto3
+import gzip
+import pytest
+import requests
+import testinfra
+from ec2instanceconnectcli.EC2InstanceConnectLogger import EC2InstanceConnectLogger
+from ec2instanceconnectcli.EC2InstanceConnectKey import EC2InstanceConnectKey
+from time import sleep
+
+postgresql_schema_sql_content = """
```

Comment:
We need to hardcode a lot of these since the setup logic lives in worker. This makes the tests brittle. One possible improvement would be to move some of the setup logic into AMI creation.

Reply:
Yeah, what I mean is to move the file templating to supabase/postgres, so that UserData only contains things that need to be variable. I'll start with adding a note for now.